test/query_test.exs

```diff
@@ -694,4 +694,25 @@ defmodule QueryTest do
     """, [])
     assert [[1], [2]] = query("SELECT raise_notice_and_return(x) FROM generate_series(1, 2) AS x", [])
   end
+
+  test "executes query in chunks", context do
+    assert transaction(fn(conn) ->
+      assert {:ok, q} = P.prepare( conn, "", "SELECT * FROM generate_series( 1, 3 )", max_rows: 1 )
+      assert {:ok, %Postgrex.Result{rows: [[ 1 ]], query: q} } = P.execute( conn, q, [] )
+      assert {:ok, %Postgrex.Result{rows: [[ 2 ]], query: q} } = P.execute( conn, q, [] )
```

Comment:
What happens if we execute a different query between these two continuation `execute/3` calls?